api4/command_test.go

```diff
@@ -423,6 +423,14 @@ func TestExecuteCommand(t *testing.T) {
 	cmdPosted := false
 	for _, post := range posts.Posts {
 		if strings.Contains(post.Message, "test command response") {
+			if post.Type != "custom_test" {
+				t.Fatal("wrong type set in slash command post")
+			}
```

Comment:
The [testify assert library](https://godoc.org/github.com/stretchr/testify/assert#Equal) lets you do checks like this in one line